index.js

```diff
@@ -25,15 +25,16 @@ export const pageIndicator = {
 };
 
 export const ribbon = {
-  base: 'py-4 px-8 border rounded-4 inline-flex last-child:mb-0',
-  info: 'border-aqua-50 bg-aqua-50',
-  success: 'border-green-100 bg-green-100',
-  warning: 'border-yellow-100 bg-yellow-100',
-  error: 'border-red-100 bg-red-100',
-  disabled: 'border-bluegray-300 bg-bluegray-300',
-  sponsored: 'border-aqua-200 bg-aqua-200',
-  neutral: 'border-gray-100 bg-gray-100',
-  outlined: 'border-bluegray-300 bg-white',
```

Comment:
Is `outlined` removed on purpose?

Reply:
yes, since there were no prop for it, it wasn't used. plus not tokens for it either